src/Umbraco.Cloud.StorageProviders.AzureBlob/AzureBlobComposer.cs

```diff
@@ -1,54 +1,45 @@
 using System;
 using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Options;
 using Umbraco.Cms.Core.Composing;
 using Umbraco.Cms.Core.DependencyInjection;
-using Umbraco.Cms.Web.Common.ApplicationBuilder;
 
 namespace Umbraco.Cloud.StorageProviders.AzureBlob
 {
     /// <summary>
     /// Automatically configures Azure Blob Storage for use on Umbraco Cloud.
     /// </summary>
     /// <seealso cref="Umbraco.Cms.Core.Composing.IComposer" />
-    public class AzureBlobComposer : IComposer
+    public sealed class AzureBlobComposer : IComposer
     {
         /// <inheritdoc />
         public void Compose(IUmbracoBuilder builder)
         {
-            if (builder == null) throw new ArgumentNullException(nameof(builder));
+            ArgumentNullException.ThrowIfNull(builder);
 
-            /* There's a bug in Microsoft.Extensions.Configuration.EnvironmentVariables @ 6.0.0 WRT env var normalization + AddEnvironmentVariables(prefix) 
-             * See https://github.com/dotnet/runtime/pull/62916, should be resolved upstream in https://github.com/dotnet/runtime/milestone/87
-             * Until then, safest thing to do is explicitly add environment variables using both prefixes.
-             * (otherwise there are issues when folks update TFM to net6) */
             var configuration = new ConfigurationBuilder()
                 .AddEnvironmentVariables("Umbraco:Cloud:")
```

Comment:
Normalizing environment variable prefixes was fixed in .NET 6.0.2 (see [diff](https://github.com/dotnet/runtime/compare/v6.0.1...v6.0.2#diff-4e48342850e65c8a886d3a29d3b67bd4b68178fcab308ca5b89e11932719fbbd)), so we should actually keep this workaround for now:

```suggestion
            // There was a bug with normalizing prefixes in .NET 6 (fixed in 6.0.2), so safest thing is to add both prefixes until the TFM is updated to net7.0
            var configuration = new ConfigurationBuilder()
                .AddEnvironmentVariables("Umbraco:Cloud:")
                .AddEnvironmentVariables("Umbraco__Cloud__")
```